README.md

```
@@ -303,6 +280,8 @@ We extend our heartfelt gratitude to all the amazing contributors who have made
                     <sub><b>Tejas Benibagde</b></sub>
                 </a>
             </td>
+		</tr>
```

Markdownlint

Column: 1
Hard tabs

(MD010, no-hard-tabs)